src/Octoshift/Commands/AbortMigration/AbortMigrationCommandHandler.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using OctoshiftCLI.Services;
+
+[assembly: InternalsVisibleTo("OctoshiftCLI.Tests")]
+
+namespace OctoshiftCLI.Commands.AbortMigration;
+
+public class AbortMigrationCommandHandler : ICommandHandler<AbortMigrationCommandArgs>
+{
+    internal int WaitIntervalInSeconds = 10;
+
+    private readonly OctoLogger _log;
+    private readonly GithubApi _githubApi;
+
+    public AbortMigrationCommandHandler(OctoLogger log, GithubApi githubApi)
+    {
+        _log = log;
+        _githubApi = githubApi;
+    }
+
+    public async Task Handle(AbortMigrationCommandArgs args)
+    {
+        if (args is null)
+        {
+            throw new ArgumentNullException(nameof(args));
+        }
+
+        await AbortRepositoryMigration(args.MigrationId);
+    }
+
+    private async Task AbortRepositoryMigration(string migrationId)
+    {
+        var abortion_state = await _githubApi.AbortMigration(migrationId);
```

Comment:
All local variables in C# are camel cased by convention, another suggestion would be to change the name to `success`:
```suggestion
        var success = await _githubApi.AbortMigration(migrationId);
```

Reply:
Woops forgot about the camel case, i like `abortionState` i think thats more descriptive of the actual value